README.md

```diff
@@ -87,11 +93,12 @@ We built Spot with this in mind. Instead of having to juggle various API format
 [![License](https://img.shields.io/npm/l/@airtasker/spot.svg)](https://github.com/airtasker/spot/blob/master/package.json)
 
 <!-- toc -->
-* [Spot](#spot)
-* [Why we built Spot](#why-we-built-spot)
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
+
```

Comment:
Is this also just an extra space? Just nit

Reply:
yup, this stuff gets butchered during the release